src/plugins/dashboard/server/index.ts

```diff
@@ -15,6 +15,9 @@ export const config: PluginConfigDescriptor<ConfigSchema> = {
     allowByValueEmbeddables: true,
   },
   schema: configSchema,
+  deprecations: ({ deprecate }) => {
+    return [deprecate('allowByValueEmbeddables', '8.6.0', { level: 'warning' })];
```

Comment:
👍

Reply:
Thanks Tina! Just to confirm, this will make this setting show up in `stack_stats.kibana.plugins.core.config.deprecatedKeys.unset` when it is unset?

Reply:
I believe so, yes.
cc @Bamieh  please correct me if I'm wrong

Reply:
If i'm not mistaken the `deprecate` helper will only log a warning. I recommend using the `unused` helper for this config 

This is the code for the deprecate helper:
https://github.com/elastic/kibana/blob/main/packages/kbn-config/src/deprecation/deprecation_factory.ts#L209

Reply:
Hey @Bamieh, sorry for the late reply! Will the `unused` helper add telemetry? If not, is there a way we can do so? One of the main reasons for deprecating this is so we can see if _any_ clusters are using it. It's unlikely that any are.

Will the unused helper throw warnings or errors on startup if a cluster happens to be configured with that key?